R/modify_level.R

```diff
@@ -1,3 +1,5 @@
+#' @param by (optional) quoted name of level \code{modify_level} uses to split-modify-combine data.
```

Comment:
s/level/variable/ - it need not be an official "fabricator level" - see eg the unit test which uses the sleep data set.